### PR TITLE
[5.4] Added parameter length in executeFunction

### PR DESCRIPTION
**Pull request by ardani**
parameter length purpose for custom length in result bindParam

### CODE REVIEW

**Comment by ardani** [comment]
hi @yajra please review my PR
thanks

**Comment by yajra** [comment]
thanks @ardani, the PR looks good to go. Will do some testing when I had the chance before merging.

File: src/Oci8/Oci8Connection.php [hunk 1]
```diff
@@ -218,7 +218,7 @@ protected function getDoctrineDriver()
      * @param int $returnType (PDO::PARAM_*)
      * @return mixed $returnType
```
Comment (yajra) [inline src/Oci8/Oci8Connection.php]: Please update the doc blocks. Thanks!

Comment (skyshilabs) [inline src/Oci8/Oci8Connection.php]: okay

Comment (ardani) [inline src/Oci8/Oci8Connection.php]: done

File: src/Oci8/Oci8Connection.php [hunk 2]
```diff
@@ -216,9 +216,10 @@ protected function getDoctrineDriver()
      * @param string $sql (mixed)
      * @param array $bindings (kvp array)
      * @param int $returnType (PDO::PARAM_*)
+     * @param int $length
      * @return mixed $returnType
      */
-    public function executeFunction($sql, array $bindings = [], $returnType = PDO::PARAM_STR)
+    public function executeFunction($sql, array $bindings = [], $returnType = PDO::PARAM_STR, $length = null)
```
Comment (yajra) [inline src/Oci8/Oci8Connection.php]: Thanks for the updates. Just realized, the default [max length](http://php.net/manual/en/function.oci-bind-by-name.php) is `-1`. Do you think it's better if set `$length = -1` or `null` is ok?

Comment (ardani) [inline src/Oci8/Oci8Connection.php]: i think null is ok 
reference [PDO::bindParam](https://github.com/JetBrains/phpstorm-stubs/blob/master/PDO/PDO.php#L1061)

**Comment by yajra** [comment]
Released on v5.4.7. Thanks!

**Pull request merged**